app/views/dashboard/code.html.haml

```diff
@@ -1,11 +1,11 @@
-.stripe.reverse#code-of-conduct
```

Comment:
@KimberleyCook I think this id was used as an anchor tag so can we add it back?

Reply:
Ah ok cool, I tried looking but couldn't find where it was used. I could see the ones on the eligibility page.